Products/CMFPlone/browser/templates/recyclebin.pt

```diff
@@ -0,0 +1,349 @@
+                                    <td>
+                                        <span class="badge rounded-pill bg-light text-dark border" tal:content="item/type">Content type</span>
+                                    </td>
+                                    <td class="text-muted small">
+                                        <i class="fas fa-folder me-1"></i>
+                                        <span tal:content="item/path">Original path</span>
+                                        <div tal:condition="python:item.get('type') == 'Discussion Item' and item.get('content_path')"
+                                             class="text-muted smaller mt-1">
+                                            <i class="fas fa-link me-1"></i> Content: 
+                                            <span tal:content="item/content_path">Content path</span>
+                                        </div>
+                                    </td>
+                                    <td>
+                                        <i class="far fa-clock me-1"></i>
+                                        <span tal:content="python:view.format_date(item['deletion_date'])">Deletion date</span>
+                                    </td>
+                                    <td>
+                                        <i class="fas fa-weight me-1"></i>
+                                        <span tal:content="python:view.format_size(item.get('size', 0))">Size</span>
+                                    </td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+                
+                <div tal:condition="items" class="card-footer bg-light">
+                    <div class="d-flex justify-content-between align-items-center flex-wrap gap-2">
+                        <div class="btn-group" role="group">
+                            <button type="submit" name="form.buttons.restore" class="btn btn-primary" i18n:attributes="value">
+                                <i class="fas fa-undo me-1"></i> Restore selected
+                            </button>
+                            <button type="submit" name="form.buttons.delete" class="btn btn-danger" i18n:attributes="value">
+                                <i class="fas fa-trash-alt me-1"></i> Delete selected
+                            </button>
+                        </div>
+                        <button type="submit" name="form.buttons.empty" class="btn btn-outline-danger" 
+                                onclick="return confirm(recyclebinEmptyMessage);" i18n:attributes="value">
+                            <i class="fas fa-trash me-1"></i> Empty recycle bin
+                        </button>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </div>
+</metal:main>
+
+<metal:js fill-slot="javascript_head_slot">
+<script>
```

Comment:
please use `pat-checklist` for that ... see https://plone.github.io/mockup/docs/external/@patternslib/patternslib/src/pat/checklist/documentation/

Reply:
done in https://github.com/plone/Products.CMFPlone/pull/4168/commits/aa2eddd11ff612031fee466f1bc456b3c7c71421